src/text-area.tsx

```diff
@@ -76,6 +76,7 @@ export class TextArea extends Component<
       ...attributes
     } = this.props;
 
+    const domAttributes = { ...attributes, maxHeight: null as number };
```

Comment:
I think we can instead use the spread operator to extract the valid DOM attributes after pulling out the invalid attributes. Otherwise just delete the maxHeight prop from domAttributes.

```suggestion
    const { maxHeight, ...domAttributes} = attributes;
```

or after copying attributes into domAttributes, use: 
```
delete domAttributes.maxHeight
```

See https://reactjs.org/warnings/unknown-prop.html

Reply:
Setting it to null removes it from the DOM, and using the delete keyword really screws with the way the JS engine optimizes things, so I went with null rather than delete. This general issue requires a much bigger refactor in UI-components.

Reply:
Strange, seems reasonable though. Thanks for the explanation!